static/app/views/replays/detail/network/index.tsx

```diff
@@ -69,22 +70,33 @@ function NetworkList({networkSpans, startTimestampMs}: Props) {
       deps,
     });
 
-  const initialRequestDetailsHeight = useMemo(
-    () => Math.max(150, window.innerHeight * 0.25),
-    []
-  );
   const {size: containerSize, ...resizableDrawerProps} = useResizableDrawer({
     direction: 'up',
-    initialSize: initialRequestDetailsHeight,
+    initialSize: Math.max(150, window.innerHeight * 0.4), // cannot depend on containerRef :(
```

Comment:
larger size from before

I tried making this relative to `containerRef` but container ref starts as undefined, then `initialSize` gets cached by the hook and doesn't update :(

I briefly looked at what it takes to change that, but i don't think it's worth the complexity.

Reply:
Would be good to add why you can't depend on containerRef in the comment